design/one-pager-stacks-security-isolation.md

```diff
@@ -0,0 +1,187 @@
+
+![Diagram of a Platform Stack Installed](./images/platform-stack-installed.png)
+
+### Environments
+
+While platform stacks are installed at the cluster level (there is only one instance of each platform stack), they can still support the concept of multiple "environments", each represented by a namespace.
+For example, an admin can create a namespace to model their production "environment" and another namespace to model their development "environment".
+All resources within each of those environments can share a common set of provider credentials.
+This namespaced "environment" approach provides the administrator with a way to collect or aggregate all the infrastructure and services associated with a given environment.
+
+#### Platform Stack Single Instance Version Limitation
+
+One caveat of this approach is that because there is only a single instance of the platform stack in the cluster, it is therefore shared across all environments.
+This means that all environments must use the same version of the stack.
+If an admin wants to try a newer version of the stack for their dev environment, it will also result in the newer version of the stack being used in the production environment as well.
+
+This limitation could be addressed in the future by allowing multiple versions of platform stacks installed per control plane, which would require additional complexity and design work.
+To get started, we will make the simplifying constraint that only a single version of a platform stack can be installed and it will be shared across all namespaces (environments).
+
+#### Environment Policies
+
+Another potential advantage of supporting multiple namespace "environments" is that each one could be assigned their own specific policy, such as default resource classes.
+The production environment could have a completely different set of default resource classes than the development environment.
+Since these environments are simply represented by namespaces, they can take on any arbitrary meaning or scope.
+For example, instead of dev vs. prod, they could represent entirely other dimensions, such as a high performance environment, a cheap environment, an AWS only environment, etc.
+
+The full design for this support of different policies per environment is out of scope of this document, but this idea has interesting potential.
+
+## Application Stacks
+
+Application stacks are expected to represent higher level "application" functionality.
+Instead of providing the infrastructure and lower level platform components for entire environments like a platform stack, they merely provide the functionality to support a single application.
+It is expected for application stacks to (indirectly) consume and build on functionality from platform stacks in the form of resource claims.
+For example, an application stack could generate a `MySQLClaim` that is fulfilled through a default resource class by a platform stack in the form of a concrete MySQL database instance.
```

Comment:
There's no contemporary resource claim named `MySQLClaim` - should this be `MySQLInstance`?